android/sdk/src/main/java/com/taobao/weex/ui/WXRenderManager.java

```diff
@@ -81,6 +93,51 @@ public WXSDKInstance getWXSDKInstance(String instanceId) {
     return statement.getWXSDKInstance();
   }
 
+    public static int getOpenGLRenderLimitValue() {
+        int maxsize ;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            EGL10 egl = (EGL10) EGLContext.getEGL();
+            EGLDisplay dpy = egl.eglGetDisplay(EGL10.EGL_DEFAULT_DISPLAY);
+            int[] vers = new int[2];
+            egl.eglInitialize(dpy, vers);
+            int[] configAttr = {
+                    EGL10.EGL_COLOR_BUFFER_TYPE, EGL10.EGL_RGB_BUFFER,
+                    EGL10.EGL_LEVEL,0,
+                    EGL10.EGL_SURFACE_TYPE, EGL10.EGL_PBUFFER_BIT,
+                    EGL10.EGL_NONE};
+            EGLConfig[] configs =new EGLConfig[1];
+            int[] numConfig =new int[1];
+            egl.eglChooseConfig(dpy, configAttr, configs,1, numConfig);
+            if (numConfig[0] ==0) {// TROUBLE! No config found.
```

Comment:
`numConfig[0] ==0` means there is something wrong with OpenGL, the following lines will cause trouble if `numConfig[0] ==0`